PickAPic/PickAPic/ViewController.h

```diff
@@ -8,7 +8,12 @@
 
 #import <UIKit/UIKit.h>
 
-@interface ViewController : UIViewController
+@interface ViewController : UIViewController <UIImagePickerControllerDelegate, UINavigationControllerDelegate>
+
+@property (strong, nonatomic) IBOutlet UIImageView *imageView;
```

Comment:
Consider putting this in the `.m`. The only time you should put things in the `.h` is when another class needs to access it.